src/riscv-unpriv-integration.adoc

```diff
@@ -0,0 +1,388 @@
+The tag is cleared if the memory region is ever written using an operation other than a store of a full capability register.
+Any caches must preserve this abstraction.
+
+==== Instructions using capability registers
+
+{cheri_base_ext_name} adds new instructions to interact with the extended registers.
+These added instructions can be split into the following categories:
+
+* Capability manipulations (e.g. <<CADD>>, <<SCBNDS>>): for security, capabilities can only be modified in restricted ways.
+  Special instructions are provided to perform these allowed operations, for example _shrinking_ the bounds or _reducing_ the permissions.
+  Any attempt to manipulate capabilities without using the instructions clears the tag, rendering them unusable for accessing memory.
+
+* Capability inspection (e.g. <<GCBASE>>, <<GCPERM>>): capability fields (for example the _bounds_ describing what addresses the capability gives access to) are stored compressed in registers and memory.
+  These instructions give convenient access to allow software to query them.
+
+* Memory access instructions (e.g. <<LC>>, <<SC>>): capabilities must be read from and written to memory atomically along with their tag.
+  Instructions are added to perform these wider accesses, allowing capability flow between the memory and the register file.
+
+[#sec_cheri_cap_mode_int_mode]
+==== {cheri_cap_mode_name} and {cheri_int_mode_name}
+Existing RV32I/RV64I instructions as well as instructions defined by unprivileged extensions are largely unmodified.
+The one difference is that a hart using {cheri_base_ext_name} instead of RV32I/RV64I as the base ISA uses capabilities for all memory accesses.
+This is called executing in {cheri_cap_mode_name}.
+In {cheri_cap_mode_name}, instructions that access memory, as well as branches and jumps are modified to check against the full capability from the base address register (e.g. `lw t0, 16(c3)`) instead of just the integer address (i.e. `lw t0, 16(x3)`).
+In some cases, they are also changed to return a full capability value, e.g. <<AUIPC>> will return the full <<pcc>> including the metadata.
+For the full list of instructions that change behavior with {cheri_base_ext_name}, see <<app_cheri_instructions>>.
+
+The new CHERI state and instructions can also be made available to a hart using RV32I/RV64I as the base ISA with full binary compatibility.
+The mechanics for this mode are described by the <<section_cheri_hybrid_ext,{cheri_default_ext_name}>> extension.
+When a hart supports CHERI but is using RV32I/RV64I as the base ISA, it is executing in {cheri_int_mode_name}.
+For instructions that have differing behavior in {cheri_cap_mode_name} and {cheri_int_mode_name} (memory access and pointer arithmetic instructions), the descriptions will list assembler syntax and behavior for both modes.
```

Comment:
I think I'd remove this sentence
maybe just write something like "Integer pointer mode allows full compatibility with existing RV32I/RV32E binaries, but with with all data memory accesses and program counters checked against capabilities"